sympy/core/tests/test_evalf.py

```diff
@@ -579,3 +579,7 @@ def test_issue_13425():
 
 def test_issue_17421():
     assert N(acos(-I + acosh(cosh(cosh(1) + I)))) == 1.0*I
+
+
+def test_issue_19774():
+    assert exp(x/2).evalf() == exp(0.5*x)
```

Comment:
Why was this testcase added? Testcase should be added only for those cases which are working.

Reply:
#19774